ssl/t1_lib.c

```diff
@@ -1135,10 +1152,10 @@ int ssl_add_clienthello_tlsext(SSL *s, WPACKET *pkt, int *al)
             return 0;
         }
         /* Copy curve ID if supported */
-        for (i = 0; i < num_curves; i++, pcurves += 2) {
+        for (i = 0; i < num_curves; i++, pcurvestmp += 2) {
```

Comment:
not for this PR, but can we maybe get rid of the network-byte-order two-entries table (both of them) and use a struct or something?